### ISSUE TITLE
Rename OAuth facade

### ISSUE THREAD

**Issue by adamwathan** [adamwathan/eloquent-oauth#76]
Conflicts with a reasonably common PHP extension.

**Comment by dusterio** [comment]
Does it matter – the alias is 'SocialAuth' anyway?

**Comment by adamwathan** [comment]
Yeah updating the docs to recommend aliasing it as `SocialAuth` was sort of a quick workaround in the mean time. I think it's a better name overall regardless, the name of the package is sort of regrettable at this point even, haha...
